### PR TITLE
Add postgres service to travis CI

### PR DESCRIPTION
**Pull request by rnestler**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,6 +2,8 @@ language: python
 sudo: false
 python:
     - 3.5
+services:
+    - postgresql
```
Comment (dbrgn) [inline .travis.yml]: Hm, will this work with `DATABASE_URL=...localhost/...`? Also, why did it work before?

Comment (rnestler) [reply .travis.yml]: > Hm, will this work with DATABASE_URL=...localhost/...?

Apparently yes. Why shouldn't it?

> Also, why did it work before?

I have no clue :/ The difference that I see is that the old build runs on Ubuntu 14.04 and the new one on Ubuntu 16.04

Comment (dbrgn) [reply .travis.yml]: > Apparently yes. Why shouldn't it?

From other CI systems I'm used to services that are started as separate containers, linked using a hosts entry. But it's possible that Travis uses different base images depending on the services chosen :slightly_smiling_face: 

> I have no clue :/ The difference that I see is that the old build runs on Ubuntu 14.04 and the new one on Ubuntu 16.04

Ok, they probably removed the server in the Ubuntu 16 setup.

Comment (rnestler) [reply .travis.yml]: If you look at the build output they have a huge master image (not even sure if in docker or just a separate VM) which includes everything. The only thing the directive does is to start the postgres server with systemctl: https://travis-ci.org/coredump-ch/interna/builds/660118016#L165

**Pull request merged**